docs(portal): sso configuration examples (#2387)
Kong/developer.konghq.com: app/dev-portal/sso.md

* Developers are auto-approved by {{site.konnect_short_name}} when using SSO to log in to the Dev Portal.
  * Kong outsources the approval process to the IdP, so access restrictions must be configured in the IdP.
* If you are using [team mappings from an IdP](/dev-portal/team-mapping/), they must come from the same IdP as your Dev Portal SSO.

> * To automatically initiate the login flow when linking developers to the Dev Portal, utilize the SSO URL: `https://$YOUR_DOMAIN.com/login/sso`

Reviewer: ```suggestion
* To automatically initiate the login flow when linking developers to the Dev Portal, use the SSO URL: `https://$YOUR_DOMAIN.com/login/sso`
```

To ensure the preview experience in the {{site.konnect_short_name}} Dev Portal Editor works correctly, configure your IdP with the following:

* Set the Sign On URL (SSO URL) to the login path of your Dev Portal's domain:
  `https://$YOUR_DOMAIN.com/login/sso`
* For SAML:
  * Set the primary **Reply URL** (Assertion Consumer Service URL) to:
    `https://$YOUR_DOMAIN.com/api/v2/developer/authenticate/saml/acs`
  * Add an additional Reply URL to support preview mode:
     `https://$YOUR_SUBDOMAIN.portal-preview.$GEO.api.konghq.com/api/v2/developer/authenticate/saml/acs`
* Allow iframe embedding of the IdP's sign-in screen:
  * For example, Okta requires [Trusted Origins](https://help.okta.com/en-us/content/topics/api/trusted-origins-iframe.htm).
    Add `https://cloud.konghq.com` as a Trusted Origin to allow login in the preview.

> ### Configuration Example

Reviewer: ```suggestion
## Dev Portal SSO configuration example
```

> Included below is an example configuration. For the example, we will utilize the following values:

Reviewer: ```suggestion
The following is an example Dev Portal SSO configuration. For the example, we will use the following values:
```

- **{{site.konnect_short_name}} Dev Portal region/geo**: `us`
- **{{site.konnect_short_name}} Dev Portal provided domain**: `abc123456789.us.kongportals.com`
- **Desired custom domain**: `www.example.com`

> #### OIDC-specific settings

Reviewer: ```suggestion
### OIDC-specific settings
```

> - **Single Sign On URL (SSO URL)**: `https://www.example.com/login/sso`

Reviewer: ```suggestion
If you're using OIDC for SSO, you'd configure your settings like the following:
- **Single Sign On URL (SSO URL)**: `https://www.example.com/login/sso`
```

>   - Also sometimes referred to as **Initiate login URI**, utilized to automatically initiate the SSO login flow.

Reviewer: ```suggestion
  - Also sometimes referred to as **Initiate login URI**, use to automatically initiate the SSO login flow.
```

- **Sign-in redirect URIs**:
  - `https://www.example.com/login`
  - `https://abc123456789.portal-preview.us.api.konghq.com/login`

- **Sign-out redirect URIs**:
  - `https://www.example.com/login`
  - `https://abc123456789.portal-preview.us.api.konghq.com/login`

> #### SAML-specific settings:

Reviewer: ```suggestion
### SAML-specific settings:
```

> - **Single Sign On URL (SSO URL)**: `https://www.example.com/api/v2/developer/authenticate/saml/acs`

Reviewer: ```suggestion
If you're using SAML for SSO, you'd configure your settings like the following:
- **Single Sign On URL (SSO URL)**: `https://www.example.com/api/v2/developer/authenticate/saml/acs`
```

>   - Also sometimes referred to as **Initiate login URI**, utilized to automatically initiate the SSO login flow.
>   - To automatically initiate the login flow when linking developers to the Dev Portal, utilize the dedicated path: `https://www.example.com/login/sso`

Reviewer: ```suggestion
  - Also sometimes referred to as **Initiate login URI**, used to automatically initiate the SSO login flow.
  - To automatically initiate the login flow when linking developers to the Dev Portal, use the dedicated path: `https://www.example.com/login/sso`
```
